models/pl_pos_tagger-0.0.1.dvc

```diff
@@ -0,0 +1,8 @@
+md5: 6ea842df6957e0f2a1c95bdb15539a0a
+outs:
+- cache: true
+  md5: 04ac1c6e90c3d0b933569957ccfa2ea3.dir
+  metric: false
+  path: models/pl_pos_tagger-0.0.1
```

Comment:
Dobrze rozumiem, to jest spakowany tagger? I nie jest stepem, bo pakujemy ręcznie? Spakowane modele umieszczałbym raczej w release, models zostawiłbym na modele będące outputem treningu.